utils/fhirpath/fhir_path_processor.bal

```diff
@@ -24,15 +24,15 @@ const string BRACKET_START = "[";
 # + fhirResource - requested fhir resource
 # + fhirPathExpression - requested fhirpath expression
 # + return - result of the fhirpath expression
-public isolated function evaluateFhirPath(map<json> fhirResource, string fhirPathExpression) returns string|json|int|float|boolean|byte|error {
+public isolated function evaluateFhirPath(json fhirResource, string fhirPathExpression) returns string|json|int|float|boolean|byte|error {
 
     // Set the root result as the first token
     map<json> result = <map<json>>fhirResource.toJson();
```

Comment:
```suggestion
    map<json> result = <map<json>>fhirResource;
```

Reply:
fixed in 4e85e156351d4fb109516d640c44557009539e84